Makefile

```diff
@@ -99,7 +99,7 @@ test:
 .PHONY: test-cockroach
 test-cockroach: cleanup-test-cockroach
 	@docker run -d --name dss-crdb-for-testing -p 26257:26257 -p 8080:8080  cockroachdb/cockroach:v20.1.1 start --insecure > /dev/null
-	go run ./cmds/db-manager/main.go --schemas_dir ./build/deploy/db-schemas/defaultdb --db_version v3.0.0 --cockroach_host localhost
+	go run ./cmds/db-manager/main.go --schemas_dir ./build/deploy/db-schemas/defaultdb --db_version v3.1.0 --cockroach_host localhost
```

Comment:
Not for this PR, but we should probably keep track of the db version in a central file.

Reply:
Added #351